src/content/docs/en/concepts/islands.mdx

```diff
@@ -84,4 +95,41 @@ Even better, you can tell Astro exactly how and when to render each component. I
 
 In Astro, it’s up to you as the developer to explicitly tell Astro which components on the page need to also run in the browser. Astro will only hydrate exactly what’s needed on the page and leave the rest of your site as static HTML. 
 
-**Islands are the secret to Astro’s fast-by-default performance story!**
+**Client islands are the secret to Astro’s fast-by-default performance story!**
+
+<ReadMore>Read more about [using JavaScript framework components](/en/guides/frameworkj-components/) in your project.</ReadMore>
+
+## Server islands
+
+Server islands are a way to move expensive or slow server-side code out of the way of the main rendering process, making it easy to combine high performance static HTML and dynamic server-generated components.
+
+Add the [`server:defer` directive](/en/reference/directives-reference/#server-directives) to any Astro component on your page to turn it into its own server island:
+
+```astro title="src/pages/index.astro" "server:defer"
+---
+import Avatar from '../components/Avatar.astro';
+---
+<Avatar server:defer />
+```
+
+This breaks up your page with smaller areas of server-rendered content that each load in parallel.
+
+Your page's main content can be rendered immediately with placeholder content, such as a generic avatar, until your island's own content is available. With server islands, having small components of personalized content does not delay the rendering of an otherwise static page.
```

Comment:
```suggestion
Your page's main content can be rendered immediately with placeholder content, such as a generic avatar until your island's own content is available. With server islands, having small components of personalized content does not delay the rendering of an otherwise static page.
```